### DISCUSSION TITLE
我看到插件支持BANGUMI简介刮削，在刮削完成后，描述中没有看到简介内容，这是发生了什么吗？

### DISCUSSION THREAD

**Discussion by X012C** [Ivanlon30000/PlayniteBangumiMetadata#4] [General]
![image](https://github.com/user-attachments/assets/c6e9e011-649c-47b6-8f72-b9403b430f2a)

**Comment by Ivanlon30000** [comment]
能提供更详细的信息吗？

如果能稳定复现，请提交issue（附带`extensions.log`文件）。
